src/main/java/frc/team3388/robot/Constants.java

```diff
@@ -0,0 +1,29 @@
+package frc.team3388.robot;
+
+public class Constants {
+    /**
+     * Which PID slot to pull gains from. Starting 2018, you can choose from
+     * 0,1,2 or 3. Only the first two (0,1) are visible in web-based
+     * configuration.
+     */
+    public static final int kSlotIdx = 0;
+
+    /**
+     * Talon SRX/ Victor SPX will supported multiple (cascaded) PID loops. For
+     * now we just want the primary one.
+     */
+    public static final int kPIDLoopIdx = 0;
+
+    /**
+     * Set to zero to skip waiting for confirmation, set to nonzero to wait and
+     * report to DS if action fails.
+     */
+    public static final int kTimeoutMs = 30;
+
+    /**
+     * PID Gains may have to be adjusted based on the responsiveness of control loop.
+     * kF: 1023 represents output value to Talon at 100%, 7200 represents Velocity units at 100% output
+     *
+     * 	                                    			  kP   kI   kD   kF          Iz    PeakOut */
+    public final static Gains kGains_Velocit = new Gains( 0.25, 0.001, 20, 1023.0/7200.0,  300,  1.00);
+}
```

Comment:
At the end we decided to use our own PID based system, so no need to implement all of their stuff. I'll show you how to use it this week.